knowledge/science/medical/pancreaticoduodenectomy/qna.yaml

```diff
@@ -0,0 +1,72 @@
+created_by: savva-k
+domain: medical
+seed_examples:
+  - question: What is a pancreaticoduodenectomy?
+    answer: >
+      Pancreaticoduodenectomy is a major surgical operation most often performed
+      to remove cancerous tumours
+
```

Comment:
Please remove these extra blank lines in this file.

Reply:
Fixed